package.json

```diff
@@ -1,25 +1,64 @@
 {
   "name": "linter-htmlhint",
-  "providedServices": {
-    "linter": {
-      "versions": {
-        "1.0.0": "provideLinter"
-      }
-    }
-  },
-  "main": "./lib/init",
   "version": "0.2.2",
-  "description": "Linter plugin for HTML, using htmlhint",
+  "private": true,
```

Comment:
Good idea, should probably include this in the other `package.json` files in the Org as well just in case.